src/views/LandingPage.vue

```diff
@@ -11,100 +11,75 @@
       </div>
       <h2 class="mb-2">Project Overview</h2>
       <p class="lead mb-3">
-        This web application is the user-facing end of a project created by
-        myself, aimed at giving newer developers a learning experience about
-        some of the processes and conventions they may encounter at their first
-        companies while not being too overwhelming in its source code. It's
-        meant to be somewhat opinionated about style, testing, and developer
-        workflow, focusing on simple concepts and top-notch execution over any
-        single triumphant "idea for an app".
+        This web application is the user-facing end of a project created by myself, aimed at giving newer developers a
```

Comment:
All this other stuff that your linter changed isn't a huge deal in this case, it's why dev shops tend to put all their developers on the same linting rules. Otherwise you have "warring" lint additions thrown into PR's everytime a different dev opens the file. You can get around this by using `git add --interactive` which will step through a files changes and let you pick only the lines you're truly interested in updating.